openshift-knative-operator/pkg/monitoring/helpers.go

```diff
@@ -270,3 +270,13 @@ func getRBACManifest() (mf.Manifest, error) {
 	rbacManifest, err := mf.NewManifest(rbacPath)
 	return rbacManifest, err
 }
+
```

Comment:
I think it is better to have the function comment which explains why this function is necessary as you explained in the PR message below.

> Can't be done via a normal deployment override without this patch because knative operator applies deployment [overrides before we inject kube-rbac-proxy](https://github.com/knative/operator/blob/6f94c4f029e102e753ea4786a4780acc0ceff42e/pkg/reconciler/common/transformers.go#L49) container at the extension side. Followed the same approach for Kafka for consistency.

Reply:
Sure.